Sample/Kitchen Sink/App/KitchenSinkApp.swift

```diff
@@ -34,6 +34,14 @@ struct KitchenSinkApp: App {
                         SampleScreenView(title: "Cards", destination: Cards())
                         SampleScreenView(title: "Inputs", destination: Inputs())
                     }
+
+                    Section(
+                        header: HeaderTitleView(title: "Demonstrations",
+                                                subtitle: "Functional tools for demonstration")
+                    ) {
+                        SampleScreenView(title: "Unit Converter", destination: UnitConverter())
+                    }
+
```

Comment:
The `.headerProminence(.increased)` modifier needs to be added to the sections to avoid the capitalisation of titles here 👍  adding in the new section went between this and the modifier for components section, so just need to add that back in 😂